maxim/apis/maxim_apis.py

```
@@ -110,47 +139,187 @@ def __init__(self, base_url: str, api_key: str):
+
+                    return response.content
+
+            except PoolError as e:
+                # Handle pool exhaustion separately with more aggressive retries
+                last_exception = e
+                pool_retry_count += 1
+                if pool_retry_count <= max_pool_retries:
+                    # Shorter delay for pool exhaustion - just waiting for connections to free up
+                    delay = 0.1 * (1.5**pool_retry_count)  # Shorter exponential backoff
+                    scribe().warning(
+                        f"[MaximSDK] Connection pool exhausted on attempt {pool_retry_count}/{max_pool_retries}: {e}. Retrying in {delay:.1f}s..."
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)